### ISSUE TITLE
i want use the 'set' command like this "set(final byte[] key, final byte[] value,  final byte[] expx, final int time)"

### ISSUE THREAD

**Issue by fengyapeng** [xetorthio/jedis#1318]
### Expected behavior

I want use reids  "set" command like this:
`set(final byte[] key, final byte[] value,  final byte[] expx, final int time)`
### Actual behavior

but only have this 

```
 public void set(final byte[] key, final byte[] value, final byte[] nxxx) {
    sendCommand(Command.SET, key, value, nxxx);
  }

  public void set(final byte[] key, final byte[] value, final byte[] nxxx, final byte[] expx,
      final int time) {
    sendCommand(Command.SET, key, value, nxxx, expx, toByteArray(time));
  }

```
### Redis / Jedis Configuration

normal
#### Jedis version:

2.8.1
#### Redis version:

3.0
#### Java version:

 1.8

**Comment by marcosnils** [comment]
Does this work for you? https://github.com/xetorthio/jedis/blob/2.8/src/main/java/redis/clients/jedis/BinaryJedis.java#L523

BTW: In Jedis master we've changed the way several commands receive there parameters. That's why you might find some methods that are missing.

**Comment by sazzad16** [comment]
Resolved by #1433
